packages/0xcert-ethereum-utils-contracts/src/contracts/permission/abilitable.sol

```diff
@@ -102,35 +96,32 @@ contract Abilitable
     uint256 _abilities
   )
     external
-    hasAbilities(ABILITY_TO_MANAGE_ABILITIES)
+    hasAbilities(SUPER_ABILITY)
   {
     addressToAbility[_target] |= _abilities;
-
-    if ((_abilities & ABILITY_TO_MANAGE_ABILITIES) == ABILITY_TO_MANAGE_ABILITIES)
-    {
-      zeroAbilityCount = zeroAbilityCount.add(1);
-    }
     emit GrantAbilities(_target, _abilities);
   }
 
   /**
    * @dev Unassigns specific abilities from specified address.
    * @param _target Address of which we revoke abilites.
    * @param _abilities Number representing bitfield of abilities we are revoking.
+   * @param _allowSuperRevoke Additional check so that you do not remove your own super ability by 
```

Comment:
`Additional check that prevents you from removing your own super ability by mistake.`